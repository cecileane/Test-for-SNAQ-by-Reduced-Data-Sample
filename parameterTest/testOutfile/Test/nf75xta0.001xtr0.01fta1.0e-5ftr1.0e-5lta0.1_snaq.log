optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=1.0e-5,
                       xtolAbs=0.001, xtolRel=0.01.
 max number of failed proposals = 75, liktolAbs = 0.1.
rootname for files: nf75xta0.001xtr0.01fta1.0e-5ftr1.0e-5lta0.1_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 60473
seed: 60473 for run 1
Sat Oct  1 17:39:49 2016

 BEGIN SNaQ for run 1, seed 60473 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 90 (failures: 75) with -loglik=126.74251 and ht_min=[0.2856,0.48942,1.51213,10.0,0.51916]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 1		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 24		 24		 2	 |	 0		 0		 --
mvtarget	 27		 27		 0	 |	 0		 0		 --
chdir		 5		 5		 1	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 33		 27		 0	 |	 8		 8		 --
Total		 90		 84		 4	 |	 8		 8		 8
Proportion	 -- 		 1.0		 0.0	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 1, -loglik of best 126.74250981975594
(3,((2,(1)#H7:::0.7143969941118264):1.5121333899447478,4):0.48941864605250424,(5,(6,#H7:::0.28560300588817356):10.0):0.5191607397368316);
---------------------
seed: 29549 for run 2
Sat Oct  1 17:39:55 2016

 BEGIN SNaQ for run 2, seed 29549 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED for not having more moves to propose: movesfail [0,25,25,4,0,61], Nmov [217,25,25,4,10000,61]
END optTopLevel: found minimizer topology at step 178 (failures: 54) with -loglik=192.10452 and ht_min=[0.49579,0.13851,0.0,9.89141,0.2498,0.0,0.0]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 1		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 40		 40		 5	 |	 0		 0		 --
mvtarget	 35		 35		 1	 |	 0		 0		 --
chdir		 23		 23		 0	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 78		 3		 1	 |	 2		 2		 --
Total		 177		 102		 8	 |	 2		 2		 2
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 2, -loglik of best 192.10452450781983
(5,6,(((2,#H7:0.0::0.49578508860087067):9.891406489878271,3):0.13851490084124649,((4,1):0.0)#H7:0.0::0.5042149113991293):0.24980417168508331);
---------------------

Sat Oct  1 17:40:01 2016
MaxNet is (3,((2,(1)#H7:::0.7143969941118264):1.5121333899447478,4):0.48941864605250424,(5,(6,#H7:::0.28560300588817356):10.0):0.5191607397368316); 
with -loglik 126.74250981975594
