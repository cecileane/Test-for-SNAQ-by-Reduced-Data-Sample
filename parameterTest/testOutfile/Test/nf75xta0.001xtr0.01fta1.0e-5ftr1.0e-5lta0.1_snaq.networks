(3,((2,(1)#H7:::0.7143969941118264):1.5121333899447478,4):0.48941864605250424,(5,(6,#H7:::0.28560300588817356):10.0):0.5191607397368316);, with -loglik 126.74250981975594 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
(3,((2,(1,#H7:::0.30904432094523904):4.35077142594708):0.6621244088712808,4):0.46446785587652334,(5,(6)#H7:::0.690955679054761):1.6808996321100789);, with -loglik 128.2708728145751
((4,(3)#H7:::0.6029268021630954):0.04257938361958681,2,(1,(6,(5,#H7:::0.3970731978369047):9.95756785494488):0.3152002841900405):0.2032590079431423);, with -loglik 268.556976074196
(3,(#H7:::0.28983236280396196,4):2.8763986484855284,(5,(6,(1,(2)#H7:::0.710167637196038):0.3526512401059713):0.19184625886854628):0.0);, with -loglik 278.28831402049417
(3,(4)#H7:::0.6311340675211077,(5,(6,(1,(#H7:::0.3688659324788923,2):4.0557304426456335):0.0884140083809805):0.4250072795665629):0.0);, with -loglik 310.3119857878371
(3,((2,(1,(6,#H7:::0.4801543734242713):0.016432790323386737):0.1469927292731806):0.07039527852157977,4):0.6013629769289841,(5)#H7:::0.5198456265757287);, with -loglik 392.33112801822074
