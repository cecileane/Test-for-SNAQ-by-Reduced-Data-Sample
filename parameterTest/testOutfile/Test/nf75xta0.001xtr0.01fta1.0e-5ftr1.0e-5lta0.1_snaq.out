(3,((2,(1)#H7:::0.7143969941118264):1.5121333899447478,4):0.48941864605250424,(5,(6,#H7:::0.28560300588817356):10.0):0.5191607397368316); -Ploglik = 126.74250981975594
 Dendroscope: (3,((2,(1)#H7):1.5121333899447478,4):0.48941864605250424,(5,(6,#H7):10.0):0.5191607397368316);
 Elapsed time: 11.624838087 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 (3,((2,(1)#H7:::0.7143969941118264):1.5121333899447478,4):0.48941864605250424,(5,(6,#H7:::0.28560300588817356):10.0):0.5191607397368316);, with -loglik 126.74250981975594
 (5,6,(((2,#H7:0.0::0.49578508860087067):9.891406489878271,3):0.13851490084124649,((4,1):0.0)#H7:0.0::0.5042149113991293):0.24980417168508331);, with -loglik 192.10452450781983
-------
